service/src/integrationTest/kotlin/fi/espoo/evaka/messaging/MessageAccountQueriesTest.kt

```diff
@@ -99,6 +103,16 @@ class MessageAccountQueriesTest : PureJdbiTest() {
         assertEquals("Testiläiset", groupAccount.daycareGroup?.name)
     }
 
+    @Test
+    fun `employee has no access to inactive accounts`() {
+        val groupAccountName = "Test Daycare - Testiläiset"
+        db.transaction { it.deactivateEmployeeMessageAccount(employeeId) }
+
+        val accounts = db.transaction { it.getMessageAccountsForUser(employee) }
+        assertEquals(1, accounts.size)
+        assertEquals(1, accounts.filter { it.name == groupAccountName }.size)
```

Comment:
Could maybe add ` assertEquals(2, db.read { it.getMessageAccountsForUser(employee) }.size)` before deactivation.